pycbc/waveform/waveform.py

```diff
@@ -110,6 +110,8 @@ def _check_lal_pars(p):
         lalsimulation.SimInspiralWaveformParamsInsertFrameAxis(lal_pars, p['frame_axis'])
     if p['side_bands']:
         lalsimulation.SimInspiralWaveformParamsInsertSideband(lal_pars, p['side_bands'])
+    if p['mode_array']:
```

Comment:
I don't know if I agree or disagree with this.

The downside is that there might be convenience functions in LAL (add all l=2 modes ... add all l=3 modes .. etc.) Okay duplicating them in PyCBC is not too difficult, but is it simpler to avoid this duplication and use the lal type natively? I mean get_td_waveform is tied to lalsimulation, so I don't see why lalsimulation types should *not* be allowed here?